substrate/cmd/bootstrap.go

```diff
@@ -49,9 +52,9 @@ func bootstrap(cmd *cobra.Command, args []string) {
 	}
 	ctx := cmd.Context()
 	start := time.Now()
-	// TODO make substrate name configurable
-	name := "test-substrate"
-	logging.FromContext(ctx).Infof("Starting bootstrap of %q", name)
+	u, err := user.Current()
+	runtime.Must(err)
+	name := fmt.Sprintf("kitctl-%s", u.Username)
```

Comment:
This might not work for CI environments, as the Username mostly will be something like `root` or `ec2-user`.

Reply:
We should customize the name in CI environments.